lib/src/bloc/genres_bloc.dart

```diff
@@ -1,22 +1,23 @@
+import 'package:rxdart/rxdart.dart';
 import 'bloc.dart';
 import 'dart:async';
 import '../models/genre_model.dart';
 import '../resources/movies_repository.dart';
 
 class GenresBloc extends Bloc {
   final _repository = MoviesRepository();
-  final _genresFetcherController = StreamController<GenreModel>();
+  final StreamController<GenreModel> _controller = BehaviorSubject();
```

Comment:
you should return the stream controller name to the first one, it was more descriptive

Reply:
Done.